__test__/satysrc/generic.saty

```diff
@@ -400,6 +400,10 @@ let () = TreeSet.size t |> String.of-int |> Debug.log in
 let t = TreeSet.remove Int.ord 10 t in
 let () = p t in
 let () = TreeSet.size t |> String.of-int |> Debug.log in
+% See https://github.com/nyuichi/satysfi-base/pull/140#discussion_r550355160
+let t = [9;4;11;2;6;10;13;1;3;5;8;12;0;7] |> List.fold-left (fun s x -> TreeSet.insert Int.ord x s) TreeSet.empty in
+let t = TreeSet.remove Int.ord 13 t in
+let () = p t in
```

Comment:
new test case for https://github.com/nyuichi/satysfi-base/pull/140#discussion_r550355160